hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -148,6 +149,18 @@ protected Object getFallback() {
     }
     return hystrixCommand.execute();
   }
+  
+  /**
+   * Generates the key for the Hystrix command.  This key is used by
+   * Hystrix in a cache, so we try and make the key as unique as possible.
+   * The default implementation uses the result of calling 
+   * {@link java.lang.Method.html#toString() toString} on the fallback Method.
+   * @param method The fallback method.
+   * @return A unique key to be used to cache the Hystrix command.
+   */
+  protected String generateCommandKey(Method method) {
```

Comment:
type is final so no need to talk about the difference between default impl and the actual impl (nor make it protected). nit.. uhh.. I think you forgot to call this method :)

ps LOVE the first two sentences of the docs..  the rest aren't needed as this will only be viewed by authors.